### PR TITLE
Fix issue #23 and more changes

### PR DESCRIPTION
**Pull request by shahoob**
Read the latest comment below

### CODE REVIEW

**Comment by shahoob** [comment]
I propose changes to `create-ink-app` which will:

- Fix issue #23
- Rework the CI workflow
- Add support for [Yarn](https://yarnpkg.com/) and [pnpm](https://pnpm.io)

It is ready for review and can be merged.
However, when merging, you should squash merge as 11 commits may clog up the commit history.

I also can remove support for pnpm if `ink` doesn't work by default with it.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,19 +0,0 @@
...
-        node_version: [10, 12]
-
-    steps:
-      - uses: actions/checkout@master
-      - name: Use Node.js ${{ matrix.node_version }}
-        uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node_version }}
-      - run: npm install
-      - run: npm test
```
Comment (mattferrin) [inline .github/workflows/test.yml:19]: Was `npm test` deleted altogether? (From workflows, I know it's still in `package.json`.)

Comment (mattferrin) [reply .github/workflows/test.yml:19]: I understand now. The `test` command is in the generated app. The generator only has a linter and it was a misnomer here.

**Review by mattferrin — COMMENTED** [review]
I didn't attempt to test this to validate it works. I didn't see anything functionally concerning and refrained from commenting regarding my stylistic opinions. It looks good.

File: .github/workflows/test.yml (hunk 1 above)
Comment (shahoob) [reply .github/workflows/test.yml:19]: You can actually see the workflow runs on my fork [here](https://github.com/shahoob/create-ink-app/actions).

**Comment by vadimdemedes** [comment]
Thanks for your contribution!

It would be great if this PR was about #23 exclusively. I'd like to not make the other changes suggested here, like changing scripts and supporting other package managers. This package provides an opinionated boilerplate, so it's up to developers to customize it further.

**Pull request closed without merging**